_P208_Nokia_LCD_5110.ino

```diff
@@ -217,32 +224,45 @@ boolean Plugin_208(byte function, struct EventStruct *event, String& string){
             argIndex = string.lastIndexOf(',');
             tmpString = string.substring(argIndex + 1);
             if (tmpString.equalsIgnoreCase(F("Clear"))){
-              addLog(LOG_LEVEL_INFO, F("Clear Display"));
+              myLog("Clear Display");
               lcd3->clearDisplay();
               lcd3->display();
             }
             if (tmpString.equalsIgnoreCase(F("blOn"))){
               success = true;
               PCONFIG(0) = 1;
-              setBacklight(event); 
+              setBacklight(event);
             }
             if (tmpString.equalsIgnoreCase(F("blOff"))){
               success = true;
               PCONFIG(0) = 0;
-              setBacklight(event); 
+              setBacklight(event);
             }
-          break;
-        }
+            break;
+          }
         }
       }
   }  // switch (function)
   return success;
 }
 
+//void myLog (String message) {
+//  addLog(LOG_LEVEL_INFO, "P208: " + message);
+//}
+void myLog(const String & message) {
+  String log;
+  if (sentlog){

```

Comment:
- The order of these 2 lines could be swapped
- You could also check in the if wether the log-level is set (when compiling against a current ESPEasy mega):
`if (sentlog && logLevelActiveFor(LOG_LEVEL_INFO)) {`

that would save a few cpu cycles and a bit of memory

Reply:
I can not use the current ESPmega (ESPEasy-mega-20220328) because of build errors...... 
I'm using ESPEasy_ESP32_mega-20211224 without build errors....